oak_containers_system_image/base_image/build-base.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+
+cd "$SCRIPTS_DIR/.."
+
+# Fix the file permissions that will be loaded into the system image, as Git doesn't track them.
+# Unfortunately we can't do it in Dockerfile (with `COPY --chown`), as that requires BuildKit.
+chmod --recursive a+rX files/
+
+docker build . --tag=oak-containers-sysimage-base:latest -f base_image/Dockerfile
+
+# We need to actually create a container, otherwise we won't be able to use `docker export` that gives us a filesystem image.
+# (`docker save` creates a tarball which has all the layers separate, which is _not_ what we want.)
```

Comment:
Do we still need docker save etc? I thought that would now happen in bazel

Reply:
I thought this was also needed for the push to succeed, but it looks like it's not.